arrow-libs/core/arrow-core/src/main/kotlin/arrow/core/Option.kt

```diff
@@ -357,22 +357,52 @@ sealed class Option<out A> {
     b: Option<B>,
     map: (A, B) -> C
   ): Option<C> =
-    zip(b, Some.unit, Some.unit, Some.unit, Some.unit, Some.unit, Some.unit, Some.unit, Some.unit) { b, c, _, _, _, _, _, _, _, _ -> map(b, c) }
+    zip(
+      b,
+      Some.unit,
+      Some.unit,
+      Some.unit,
+      Some.unit,
+      Some.unit,
+      Some.unit,
+      Some.unit,
+      Some.unit
+    ) { b, c, _, _, _, _, _, _, _, _ -> map(b, c) }
 
   inline fun <B, C, D> zip(
     b: Option<B>,
     c: Option<C>,
     map: (A, B, C) -> D
   ): Option<D> =
-    zip(b, c, Some.unit, Some.unit, Some.unit, Some.unit, Some.unit, Some.unit, Some.unit) { b, c, d, _, _, _, _, _, _, _ -> map(b, c, d) }
```

Comment:
Looks like these changes are just new formatting? I don't have a preference but just thought I'd mention it

Reply:
yeah, it's only reformatting. it kept doing that everytime i did a reformat code through intellij. My intellij seems to have picked up everything in editor config but the line length settings..